docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks.
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions.
+It does so by automatically reconstructing its previous state.
+Your application logic can be recovered, replayed, or paused from any previous point in your application, ensuring continuity even after failures.
+
+## Temporal Application: The Building Blocks
+
+In a Temporal application, the code you write is made up of Workflows and Activities. 
+
+### Workflow
+
+Conceptually, a workflow is a sequence of steps. 
+You've likely encountered workflows in your daily life, whether it's:
+- Using a mobile app to transfer money
+- Purchasing concert tickets
+- Booking a vacation
+- Ordering a meal delivery
+- Filing an expense report
+- Creating a new employee onboarding process
+
+In Temporal, a Workflow defines the steps your application takes to complete a task, much like a blueprint for your application. 
+It’s your business logic, written in code, outlining each step in your process. 
+This could be moving money between bank accounts, processing orders, deploying cloud infrastructure, or training an AI model. 
+
+Temporal isn’t a no-code workflow engine. 
+It is **Workflows-as-Code**. 
+Instead of dragging and dropping steps in a visual interface, you write your Workflows in code in your favorite programming language, code editor, and other tools. 
+This gives you full control and flexibility over your business processes, allowing you to build exactly what you need. 
+
+Workflows are resilient and stateful. 
+With Temporal, they are built to handle problems smoothly. 
+They can pause, pick up where they left off, or try again automatically. 
+If a failure or a crash happens, this ensures that your business processes keep running without interruptions.
+
+### Activities
+Activities are the individual tasks within your Workflow.
+
+They handle jobs that interact with the outside world, like sending an email, making a network request, writing to a database, or calling an API. 
+These tasks are where issues might occur, such as network failures or service errors. 
+Activities are pieces of code, either functions or methods depending on the programming language, that perform these tasks. 
+Activities are invoked in Workflow code. 
+
+If an Activity fails, Temporal automatically retries it based on your configuration. 
+You have full control over how often and how many times these retries should happen. 
+You can also define specific timeouts for each Activity, triggering a retry if the task takes too long to complete.
```

Comment:
Line 66 is basically the same as line 65. When you say, "You can also.." but say the same thing, it might be redundant. I would remove line 65 and just keep line 66.

Reply:
Combined them!